R/pxweb_split_query.R

```diff
@@ -23,8 +23,10 @@ pxweb_split_query <- function(pxq, px, pxmd){
   pxqd <- pxweb_query_dim(pxq)
   # Get variables that can be split
   pxqds <- pxweb_query_dim_splittable(pxq, pxmd)
-  mxv <- px$config$max_values_to_download
   
+  mxv <- px$config$max_values_to_download
+  options("mxv"=mxv)
```

Comment:
Sorry. This is still an issue. I think it might pass CRAN, but I should fix it in a better way. please remove this line (28) and let us have a NOTE for now. Then Ill fix it later today.